### PR TITLE
Fix error handling, and other minor things

### PR DESCRIPTION
**Pull request by garyo**
This is my first PR for godot-python; hope it's useful!

### CODE REVIEW

File: pythonscript/embedded/godot/hazmat/ffi/script.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 )
 from godot.bindings import Dictionary, Array
 
+# Set to True to show script loading progress
+verbose = True
```
Comment (touilleMan) [inline pythonscript/embedded/godot/hazmat/ffi/script.py]: I would be better to provide this flag as a godot-controlled parameter (see https://github.com/touilleMan/godot-python/blob/66c9a6b42cefea70a13bd5be2f147dbdcfa480fb/pythonscript/embedded/godot/hazmat/ffi/init.py#L45)

File: pythonscript/embedded/godot/hazmat/ffi/script.py [hunk 2]
```diff
@@ -96,6 +98,7 @@ def _build_property_info(prop):
 @ffi.def_extern()
 def pybind_script_init(handle, path, source, r_error):
     path = godot_string_to_pyobj(path)
+    if verbose: print("Loading python script from %s", path)
```
Comment (touilleMan) [inline pythonscript/embedded/godot/hazmat/ffi/script.py]: PEP8 would prefer to have this on two lines ;-)

**Comment by touilleMan** [comment]
Thank you for this PR, this is excellent work (I'm especially happy with the changes you made to the scons files, given I had hard time with this build system) !

Sorry for taking time to merge this, but as you may know from discord I'm currently in vacation in Thailand with few access to my computer...

Regarding the unittests not passing, it seems there is an error in the dependencies resolution:
```
gcc-7 -o pythonscript/libpythonscript.so -m64 -Wl,-rpath,'$ORIGIN/bin' -shared pythonscript/pythonscript.os pythonscript/cffi_bindings.gen.os -Lplatforms/x11-64/pypy3-v5.10.0-linux64/bin -lpypy3-c platforms/gdnative/libgdnative_wrapper_code.x11.opt.64.a
```
Should not be executed before `platforms/gdnative/libgdnative_wrapper_code.x11.opt.64.a` is downloaded.
I would suspect this is due to your replacement of `env['foo']` into `'$foo'`

**Comment by garyo** [comment]
Thanks -- I'm looking at the dep resolution thing now.

**Comment by touilleMan** [comment]
you rock :guitar: :tada:

**Comment by touilleMan** [comment]
btw I've added you to the authors (see 2e9ceca49e64a2728ee46fc5156b819e8739bbaf), tell me if you want me to add your name/change the pseudo or anything ;-)

**Comment by garyo** [comment]
Sure, my name is Gary Oberbrunner. Thanks for asking!

**Comment by touilleMan** [comment]
I've updated the author.txt 😛

**Pull request merged**